pkg/controller/directvolumemigration/task.go

```diff
@@ -244,13 +244,15 @@ func (t *Task) Run() error {
 			return liberr.Wrap(err)
 		}
 	case CreateRsyncConfig:
-		err := t.createRsyncConfig()
+		created, err := t.createRsyncConfig()
```

Comment:
Reading the code I'm not sure if this boolean gives us anything worthwhile. If `err` is `nil` then `created` is `true`. I would just keep it as a func that just returns an error unless I'm missing something.

Reply:
Boolean is only to determine if the secret exists or not

Reply:
NIT: rename to `continue` then since this bool returns true when are okay to continue to the next step. It could be false and also create some of the rsync config.